.circleci/config.yml

```
@@ -269,31 +230,70 @@ jobs:
               -destination 'platform=iOS Simulator,name=iPhone 11' \
               -scheme ApplePlatformsIntegrationMobileTests
           working_directory: test/ApplePlatformsIntegrationTestApp
+
+  build-apple-runtime:
+    <<: *apple_defaults
+    steps:
+      - checkout
+      # TODO: Use a better cache key to avoid rebuilding whole framework
+      # when it is not necessary
+      - restore_cache:
+          key: v3-repo-{{ .Environment.CIRCLE_SHA1 }}
+      - run:
+          name: Set up workspace
+          command: mkdir -p /tmp/hermes/output
+      - run:
+          name: Install dependencies
+          command: |
+            brew install cmake
```

Was this intentional?

```suggestion
            brew install cmake ninja
```